docs/single_member_restoration.md

```diff
@@ -0,0 +1,15 @@
+## Restoration of a single member in multi-node etcd-backup-restore cluster.
+
+**Note**:
+Please note here we are proposing the solution to only single member restoration within a cluster not the quorum loss scenario(when majority of members within a cluster fail).
+
+### Motivation:
+If a single etcd member within a multi-node etcd cluster goes down due to PVC corruption(invalid data-dir) then it needs to be brought back. But it can't be restore from the snapshots present in storage container as you can't restore a part of whole cluster.
```

Comment:
```suggestion
If a single etcd member within a multi-node etcd cluster goes down due to PVC corruption (invalid data-dir) then it needs to be brought back. Unlike in the single-node case, a member of a multi-node cluster can't restored from snapshots because of a metadata mismatch that prevents to run as the same member again.
```

IIUC 🙂

Reply:
> because of a metadata mismatch that prevents to run as the same member again.

yes, you are correct but I don't want to go details as it might be ambiguous for a person who is not very familiar with etcd .... doubts like "Why snapshots(snapshot of whole cluster) not holding the metadata, why metadata mismatch? Can't we store metadata somewhere else and restore from snapshots and then update the member with correct metadata ?" 😅 .... these are some possibilities of a doubts.
May be I will try to add some points of `metadata mismatch` but I don't want to remove `as you can't restore a part of whole cluster`

Reply:
Both sounds quite ambiguous as one who knows etcd a little understands that every member holds the data of the entire cluster, so by saying "restore a part of whole cluster" I got confused. I don't have a strong opinion but maybe you can find a better phrasing.

Reply:
Updated .PTAL